libs/use-publisher/src/constants.ts

```diff
@@ -2,6 +2,7 @@ import { Bitrate } from './types';
 
 export const bitrateList: Bitrate[] = [
   { name: 'Auto', value: 0 },
+  { name: '4 Mbps', value: 4_000 },
```

Comment:
@dubeyShivank although I'm able to visually test this change, I was wondering whether there is a way for me to make sure we're streaming in the bitrate?

Reply:
@dubeyShivank although I'm able to visually test this change, I was wondering whether there is a way for me to make sure we're streaming in the bitrate?

Reply:
Yup, you can check stats on the viewer end. The total bitrate should be under 4 Mbps